DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -306,10 +308,10 @@ private IActionResult AddAdminFieldPostSave(int customisationId, AddAdminFieldVi
             }
 
             if (courseAdminFieldsService.AddAdminFieldToCourse(
-                customisationId,
-                model.AdminFieldId!.Value,
-                model.OptionsString
-            ))
+                    customisationId,
+                    model.AdminFieldId!.Value,
+                    model.OptionsString
+                ))

```

Comment:
This indentation change seems wrong

Reply:
The formatter did this. How would you prefer it? I'm tempted to put it all on one line:
```
if (courseAdminFieldsService.AddAdminFieldToCourse(customisationId, model.AdminFieldId!.Value, model.OptionsString))
{
```
Alternatively, I can assign to a variable first and use that in the `if`

Reply:
Hmm weird, I opened the file myself and ran the formatter, and it changed it back to the way it was. I'd just try running the formatter again but if it keeps wanting it like this it's not a huge deal. Don't think we need it in a variable, and having it on one line is probably too long. :)